apis/kueue/webhooks/workload_webhook_test.go

```diff
@@ -398,3 +535,153 @@ func TestValidateWorkloadUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestListRangeGetter(t *testing.T) {
```

Comment:
We shouldn't need this test once we use the controller-runtime client.

Reply:
removed